05-fundamental-development-workflows.Rmd

```diff
@@ -59,7 +59,7 @@ Recommendations:
 - Call `usethis::create_package('yourPackageName')`
 - In RStudio, do File > New Project > New Directory > R Package. This ultimately calls usethis::create_package(), so really there’s just one way.
 
-(I prefer to use RStudio - show using create_package from bash)
+~~(I prefer to use RStudio - show using create_package from bash)~~
```

Comment:
Do you want to keep this here but crossed out? If you aren't going to talk about it, be bold; update the notes to what you want them to be 😊

Reply:
I don't want future cohorts to be limited by my tentative grasp of Bash -- I think keeping it here crossed out so that someone (maybe even someone from my cohort) can comment on it in the future would be a nice middle ground.

Reply:
Ok, works for me!